pkg/agent/route/route_linux.go

```diff
@@ -1245,6 +1258,59 @@ func (c *Client) initWireguard() {
 	}
 }
 
+func (c *Client) initNodeLatency() {
+	gateway := c.nodeConfig.GatewayConfig.Name
+	if c.networkConfig.TrafficEncapMode.IsNetworkPolicyOnly() {
+		gateway = c.networkConfig.TransportIface
+	}
+	icmpZero := int32(0)
+
+	if c.networkConfig.IPv6Enabled {
+		antreaInputChainRulesIPv6 := []string{
+			iptables.NewRuleBuilder(antreaInputChain).
+				MatchInputInterface(gateway).
+				MatchICMP(&icmpZero, &icmpZero, iptables.ProtocolIPv6).
+				SetComment("Antrea: allow ICMP ingress packets from NodeLatencyMonitor").
+				SetTarget(iptables.AcceptTarget).
+				Done().
+				GetRule(),
+		}
+		antreaOutputChainRulesIPv6 := []string{
+			iptables.NewRuleBuilder(antreaOutputChain).
+				MatchOutputInterface(gateway).
+				MatchICMP(&icmpZero, &icmpZero, iptables.ProtocolIPv6).
+				SetComment("Antrea: allow ICMP egress packets from NodeLatencyMonitor").
+				SetTarget(iptables.AcceptTarget).
+				Done().
+				GetRule(),
+		}
+		c.nodeLatencyMonitorIPTablesIPv6.Store(antreaInputChain, antreaInputChainRulesIPv6)
+		c.nodeLatencyMonitorIPTablesIPv6.Store(antreaOutputChain, antreaOutputChainRulesIPv6)
+	}
+	if c.networkConfig.IPv4Enabled {
```

Comment:
I see the rules for type 0 (echo reply) but not for 8 (echo request)

similar comment for IPv6 above